### ISSUE TITLE
KCom2: Currency Symbol and Position should be set in store setting

### ISSUE THREAD

**Issue by trungjc** [Kademi/kademi-dev#6694]
Currently, the Currency symbol and position is setting up in components setting. But I think It should be set in store setting because  the currency symbol is identical in site

**Comment by bradmac** [comment]
Yep, i agree, but we'll come back to this later when we have time. For now component settings are fine.
